src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.ts

```diff
@@ -314,33 +314,82 @@ export class HearingTimingComponent extends RequestHearingPageFlow implements On
     }
   }
 
+  private isDateValidFormat(formGroup: FormGroup, id: string): boolean {
+    const dayControl = formGroup.get(`${id}_day`);
+    const monthControl = formGroup.get(`${id}_month`);
```

Comment:
As part of Ben's accessibility work you may find that these Id's change. Fine for now